src/core.e/LeastMost.pm6

```diff
@@ -0,0 +1,111 @@
+constant Least = Any.new;
+constant Most  = Any.new;
+
+$*LEAST = Least;
+$*MOST  = Most;
+
+multi sub infix:<cmp>(Least, Least --> Same) { }
```

Comment:
Bear with me here. This will work as long as these have the same value as the constants. So I don't really see what this is buying you. If you initialize it to -Inf, +Inf you get the same for free...